README.md

```diff
@@ -281,6 +281,16 @@ jscodeshift(ast).findIdentifiers().logNames();
 jscodeshift(ast).logNames(); // error, unless `ast` only consists of Identifier nodes
 ```
 
+### Passing options to [recast]
+You may want to change some of the output settings (like setting `'` instead of `"`). This can be done by passing config options to [recast].
```

Comment:
Looks like elsewhere in this file we're hard-wrapping long lines. Care to match that?